Tasks/Common/azure-arm-rest/azure-arm-network.ts

```diff
@@ -14,7 +14,7 @@ export class NetworkManagementClient extends azureServiceClient.ServiceClient {
     constructor(credentials: msRestAzure.ApplicationTokenCredentials, subscriptionId, baseUri?: any, options?: any) {
         super(credentials, subscriptionId);
 
-        this.apiVersion = '2016-09-01';
+        this.apiVersion = (credentials.isAzureStackEnvironment) ? '2015-06-15' : '2016-09-01' ;
```

Comment:
Azure Stack is onPrem Azure environment. It's API versions are not is sync with Azure API versions.